src/web/lib/renderElement.tsx

```diff
@@ -733,6 +733,7 @@ const bareElements = new Set([
 	'model.dotcomrendering.pageElements.SubheadingBlockElement',
 	'model.dotcomrendering.pageElements.TextBlockElement',
 	'model.dotcomrendering.pageElements.InteractiveContentsBlockElement',
+	'model.dotcomrendering.pageElements.InteractiveBlockElement',
```

Comment:
What does adding this here mean? The comment "bareElements is the set of element types that don't get wrapped in a Figure for most article types." seems odd since it _is_ now wrapped in a figure?

Reply:
@OllysCoding yeah, elements which wrap themselves in a figure already don't need one added here. I've pushed an update to improve this comment for clarity.